src/commands/deploy.ts

```diff
@@ -0,0 +1,15 @@
+import { Command } from '@oclif/command';
+
+import { GenerateDeployFlags } from './generate';
+
+export default class Deploy extends Command {
+	static description =
+		'Generate Deploy Spec from environment configuration and deploy';
+
+	static flags = GenerateDeployFlags;
+
+	async run() {
+		const { flags } = this.parse(Deploy);
+		return true;
```

Comment:
This command does nothing at this time...?

Reply:
Yes!  [more info here](https://github.com/balena-io/katapult/pull/32#discussion_r293407997)